doc/classes/EditorSettings.xml

```diff
@@ -677,9 +677,86 @@
 		<member name="interface/theme/corner_radius" type="int" setter="" getter="">
 			The corner radius to use for interface elements (in pixels). [code]0[/code] is square.
 		</member>
+		<member name="interface/theme/custom_dark_preset/accent_color" type="Color" setter="" getter="">
+			The color to use for "highlighted" user interface elements in the editor (pressed and hovered items).
+		</member>
+		<member name="interface/theme/custom_dark_preset/additional_spacing" type="float" setter="" getter="">
+			The spacing to add for buttons and list items in the editor (in pixels). Increasing this value is useful to improve usability on touch screens, at the cost of reducing the amount of usable screen real estate.
+		</member>
+		<member name="interface/theme/custom_dark_preset/base_color" type="Color" setter="" getter="">
+			The base color to use for user interface elements in the editor. Secondary colors (such as darker/lighter variants) are derived from this color.
+		</member>
+		<member name="interface/theme/custom_dark_preset/border_size" type="int" setter="" getter="">
+			The border size to use for interface elements (in pixels).
+		</member>
+		<member name="interface/theme/custom_dark_preset/contrast" type="float" setter="" getter="">
+			The contrast factor to use when deriving the editor theme's base color (see [member interface/theme/base_color]). When using a positive values, the derived colors will be [i]darker[/i] than the base color. This contrast factor can be set to a negative value, which will make the derived colors [i]brighter[/i] than the base color. Negative contrast rates often look better for light themes.
+		</member>
+		<member name="interface/theme/custom_dark_preset/corner_radius" type="int" setter="" getter="">
+			The corner radius to use for interface elements (in pixels). [code]0[/code] is square.
+		</member>
+		<member name="interface/theme/custom_dark_preset/custom_theme" type="String" setter="" getter="">
+			The custom theme resource to use for the editor. Must be a Godot theme resource in [code].tres[/code] or [code].res[/code] format.
+		</member>
+		<member name="interface/theme/custom_dark_preset/draw_extra_borders" type="bool" setter="" getter="">
+			If [code]true[/code], draws additional borders around interactive UI elements in the editor. This is automatically enabled when using the [b]Black (OLED)[/b] theme preset, as this theme preset uses a fully black background.
+		</member>
+		<member name="interface/theme/custom_dark_preset/icon_and_font_color" type="int" setter="" getter="">
+			The icon and font color scheme to use in the editor.
+			- [b]Auto[/b] determines the color scheme to use automatically based on [member interface/theme/base_color].
```

Comment:
```suggestion
			- [b]Automatic (Follow System Theme)[/b] determines the color scheme to use automatically based on [member interface/theme/base_color].
```

Reply:
This value is determined by the theme colors themselves rather than the system theme, and I've considered it as part of the theme itself as it's included in the presets.

Reply:
Apologies, I pointed to the wrong description.